tests/data/reports/sample_mismatch.mlst.json

```diff
@@ -0,0 +1,7 @@
+{
+    "sample_mismatch": {
```

Comment:
Where is this file used since I don't see it mentioned in this PR?

Reply:
My bad. This was another file I was using for local testing that was accidentally added.

Reply:
Removed in 61c2c927b5dc422986c760f5b1e5fa9aa46e3ff0